openrtb-core/src/main/protobuf/openrtb.proto

```diff
@@ -1519,6 +1543,10 @@ message BidResponse {
       // Creative ID to assist with ad quality checking.
       optional string crid = 10;
 
+      // The taxonomy in use.
+      // Refer to the enum com.iabtechlab.adcom.v1.enums.CategoryTaxonomy for values.
+      optional int32 cattax = 30 [default = 1];
```

Comment:
is 30 intentional here? it could be 28th

Reply:
Good spot, and yes, it's to match Google's version as we aim to merge to one version (https://developers.google.com/authorized-buyers/rtb/downloads/openrtb-proto).